.husky/pre-commit

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-cd mw-webapp
-pnpm lint-staged
+FRONTEND="mw-webapp/"
+SERVER="mw-server/"
+
+if git diff --name-only --cached | grep $FRONTEND;
+  then
+    echo "FRONTEND FOLDER CHANGED --> starting husky..."
+    cd mw-webapp
+    pnpm lint-staged
+    pnpm dlx concurrently \
```

Comment:
we can intall concurrently and  in this case we don't need to load each evety time (get rid of dlx)